### PR TITLE
Add  jammed state support for MQTT lock

### PR DESCRIPTION
**Pull request by jbouwh**
## Proposed change

Add support for a `jammed` motor state on MQTT lock.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/86010
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- x ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

File: source/_integrations/lock.mqtt.markdown [hunk 1]
```diff
@@ -145,6 +145,14 @@ json_attributes_topic:
   description: The MQTT topic subscribed to receive a JSON dictionary payload and then set as sensor attributes. Usage example can be found in [MQTT sensor](/integrations/sensor.mqtt/#json-attributes-topic-configuration) documentation.
   required: false
   type: string
+motor_state_topic:
+  description: The MQTT topic subscribed to receive motor state updates. It accepts states the `state_jammed`, `state_ok`.
```
Comment (Nardol) [inline source/_integrations/lock.mqtt.markdown]:
```suggestion
  description: The MQTT topic subscribed to receive motor state updates. It accepts the states `state_jammed` and `state_ok`.
```
Only two inverted words and isn't it better to replace the comma by "and" for a list of two?

**Review by Nardol — CHANGES_REQUESTED** [review]

File: source/_integrations/lock.mqtt.markdown (hunk 1 above)
Comment (jbouwh) [reply source/_integrations/lock.mqtt.markdown]: The motor state topic is removed after discussing this with @emontnemery. So we are going to use a single state topic, like you suggested at first.

File: source/_integrations/lock.mqtt.markdown [hunk 2]
```diff
@@ -223,7 +233,7 @@ unique_id:
    required: false
    type: string
 value_template:
-  description: "Defines a [template](/docs/configuration/templating/#using-templates-with-the-mqtt-integration) to extract a value from the payload."
+  description: Defines a [template](/docs/configuration/templating/#using-templates-with-the-mqtt-integration) to extract a state value from the payload. to `state_topic`
```
Comment (Nardol) [inline source/_integrations/lock.mqtt.markdown]:
```suggestion
  description: Defines a [template](/docs/configuration/templating/#using-templates-with-the-mqtt-integration) to extract a state value from the payload.
```

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

File: source/_integrations/lock.mqtt.markdown [hunk 3]
```diff
@@ -194,27 +194,32 @@ retain:
...
   default: LOCKED
 state_locking:
-  description: The payload sent to by the lock when it's locking.
+  description: The payload sent to `state_topic` by the lock when it's locking.
   required: false
   type: string
   default: LOCKING
 state_topic:
-  description: The MQTT topic subscribed to receive state updates.
+  description: The MQTT topic subscribed to receive state updates. It accepts states configured with `state_jammed`, `state_unjammed`, `state_locked`, `state_unlocked`, `state_locking` or `state_unlocking`.
```
Comment (jbouwh) [inline source/_integrations/lock.mqtt.markdown]:
```suggestion
  description: The MQTT topic subscribed to receive state updates. It accepts states configured with `state_jammed`, `state_locked`, `state_unlocked`, `state_locking` or `state_unlocking`.
```

Comment (jbouwh) [reply source/_integrations/lock.mqtt.markdown]: Missed this one

**Review by frenck — APPROVED** [review]

**Pull request merged**